reframe/core/pipeline.py

```diff
@@ -1124,6 +1124,24 @@ def check_performance(self):
             return
 
         with os_ext.change_dir(self._stagedir):
+
+            # Check if default reference perf values are provided and
+            # store all the variables  tested in the performance check
+            is_default_present = False
+            variables = set()
+            for system, perf_data in self.reference.items():
+                variables.add((system.split(":")[-1], perf_data[-1]))
+                if system.split(":")[0] == '*':
+                    is_default_present = True
+                    break
+
+            if is_default_present is False:
```

Comment:
Change this to `if not is_default_present` or change variable to something that when it is `True` then you need to handle the missing default case.